firmware/application/apps/ui_recon.cpp

```diff
@@ -399,7 +417,14 @@ ReconView::ReconView(NavigationView& nav)
                   &button_dir,
                   &button_restart,
                   &button_mic_app,
-                  &button_remove});
+                  &button_remove,
+                  &(*record_view)});
```

Comment:
It's already a pointer, just pass it. `record_view` or if you make it a unique_ptr, then `record_view.get()`